hmpps-kotlin-spring-boot-autoconfigure/src/test/kotlin/uk/gov/justice/hmpps/kotlin/customize/ResourceServerConfigurationCustomizerTest.kt

```diff
@@ -23,7 +24,7 @@ class ResourceServerConfigurationCustomizerTest {
     fun `should ignore default unauthorized request paths`() {
       val customizer = ResourceServerConfigurationCustomizer.build {
         unauthorizedRequestPaths {
-          includeDefaults(false)
+          includeDefaults = false
```

Comment:
Also sneaked in a change to use assignment rather than calling functions in the DSL body. Just feels more DSL-like.